gradio/__init__.py

```diff
@@ -4,6 +4,7 @@
 import gradio.inputs as inputs
 import gradio.outputs as outputs
 import gradio.processing_utils
+import gradio.templates as Templates
```

Comment:
This has the effect of allowing us to import `gr.templates.Sketchpad` as `gr.Templates.Sketchpad`. Why is that useful? 

Instead I suggest making the templates first order components, just like components:

`from gradio.templates import Sketchpad, Webcam, ...`

Reply:
we do not import components from gradio.components, we import them from gradio.

it is benefical because it feels like a class and enables usage of this kind:
```
from gradio import Templates
Templates.TextArea()
```
We could also go with your suggestion however I like this one more.

Reply:
> it is benefical because it feels like a class and enables usage of this kind:

I don't think this is standard usage. You shouldn't import a Python module as a class

> we do not import components from gradio.components, we import them from gradio.

Yeah, let's do the same with templates. So that we can do `gr.Sketchpad`, `gr.Webcam`, etc.

Reply:
> it is benefical because it feels like a class and enables usage of this kind:

I don't think this is standard usage. You shouldn't import a Python module as a class

> we do not import components from gradio.components, we import them from gradio.

Yeah, let's do the same with templates. So that we can do `gr.Sketchpad`, `gr.Webcam`, etc.

Reply:
> it is benefical because it feels like a class and enables usage of this kind:

I don't think this is standard usage. You shouldn't import a Python module as a class

> we do not import components from gradio.components, we import them from gradio.

Yeah, let's do the same with templates. So that we can do gr.Sketchpad, gr.Webcam, etc.